File: 1-Beginner/1009/bee1009.java
// 1009 - Salário com Bônus
import java.io.IOException;
import java.util.Scanner;

class bee1009 {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String nome = scan.nextLine();
        double salario = scan.nextDouble(), vendas = scan.nextDouble();

        System.out.printf("TOTAL = R$ %.2f\n", salario + vendas*0.15);

        scan.close();
    }
}

File: 1-Beginner/1009/bee1009.c
// 1009 - Salário com Bônus
#include <stdio.h>
#include <stdlib.h>

int main(void) {

    char* nome = malloc(sizeof(char)*15);
    fgets(nome, 15, stdin);

    double salario, vendas;
    scanf("%lf%lf", &salario, &vendas);

    printf("TOTAL = R$ %.2f\n", salario + vendas*0.15);

    free(nome);

    return 0;
}